teaching_platform/users/controller.py

```diff
@@ -25,17 +26,20 @@ def get_user_roles(user):
 @user_controller_bp.route("/users", methods=["POST"])
 def add_user():
     request_data = request.get_json()
-    username, password = request_data.get("username"), request_data.get("password")
+    username, password, email = request_data.get("username"), request_data.get("password"), request_data.get("email")
 
-    if not username or not password: return "Incorrect json body", 400
+    if not username or not password or not email: return "Incorrect json body", 400
```

Comment:
maybe `if not all((username, password, email))`?

Reply:
Done.